CHANGELOG/CHANGELOG-2.6.2.md

```diff
@@ -0,0 +1,268 @@
+# Release notes for v2.6.2
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.6.1
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Revert of #214, node-driver-registrar will create the path specified by `--kubelet-registration-path` ([#247](https://github.com/kubernetes-csi/node-driver-registrar/pull/247), [@mauriciopoppe](https://github.com/mauriciopoppe))
+
+## Dependencies
```

Comment:
I don't see any changes that updated dependencies in the repo. Do you know where this is coming from?

Reply:
for patch releases, our docs have a different command to use:
```
GITHUB_TOKEN=<token> release-notes \
  --discover=patch-to-latest \
  --branch=release-1.1 \
  --org=kubernetes-csi \
  --repo=external-provisioner \
  --required-author="" \
  --markdown-links \
  --output out.md
```